### DISCUSSION TITLE
Welcome to the h4cker repo Discussions!

### DISCUSSION THREAD

**Discussion by santosomar** [The-Art-of-Hacking/h4cker#59] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ckza** [comment]
Technical trainer from control system background busy with SCOR preparation 😏

**Reply by santosomar** [reply]
Very nice! Hope you are doing well.

**Comment by KEthHacker** [comment]
IoT Network Security Engineer. Currently working on my CCIE SEC.

**Reply by santosomar** [reply]
Awesome! Good luck with the exam(s).

**Reply by christinelynnb** [reply]
Hi @KEthHacker! I am currently searching for a CCNP/CCIE Network Engineer to work on a NCIS project at Quantico. Do you have a top secret clearance and live in VA? If so, please drop me a note! christine@advancedrecruitingsolutions.com
Good luck on your exam!
-Christine

**Comment by bilalali18** [comment]
Hey. I am new in Cyber security space and wanted ask the community if there are any projects out there I can work on in linux administration.

**Reply by KEthHacker** [reply]
What are you looking for? Tools, Labs, information about hacking Labs, all of them?

**Reply by Usafod** [reply]
Hi, would like to have all of them.

**Reply by akeem-eng** [reply]
I will suggest , you start from your personal system,  install  virtual machine, then install Linux to learn commands.

**Comment by shonibarej** [comment]
Hello everyone. I'm on a journey to become a cybersecurity expert. I want to be part of this great community and will be happy to contribute to this project, however I don't how to. Please let me how I can be more involve.

**Comment by Netninja001** [comment]
**I am new to the group, and a beginner in the field of cyber security I wanna become a network security, plz I need an advice cuz am new to the field thank you and waiting for your reply**

**Reply by DarkHandYsmira** [reply]
You are in a good place my dear fellow. There is in our midst a Master of the trade who generously and humbly advises and instructs others how to riposte the manifold wickedness of the human heart. Be sure to share your own tips and advice if you have any. I hope you persevere and find solid purpose on your journey.

**Comment by 24a31a1240** [comment]
hello, i just started learning cyber security i am familier with linux and a little bit of networking , i am aiming to be a penetration tester i am little confused what to do next ,how to reach my goal i started hackthebox and try hack me but they costs more i cant effort them , i am stuck where to learn how to learn can any one give me a push and help me out and also can any one tell where can i get the info of new released tools so that i can stay updated

**Reply by santosomar** [reply]
Hi!

first of all, don’t feel stuck. Almost everyone feels confused at the beginning of cybersecurity.

Since you already know some Linux and networking, that’s a good start. For penetration testing, I’d suggest building your basics step by step instead of jumping straight into tools.

Focus on this path:

1. Networking basics
    Learn TCP/IP, DNS, HTTP/HTTPS, ports, protocols, subnetting, and how the internet works.
2. Linux and scripting
    Get comfortable with the terminal, permissions, processes, services, bash scripting, and a little Python.
3. Web basics
    Learn how websites work: HTML, JavaScript, requests, cookies, sessions, APIs, and databases.
4. Security fundamentals
    Study common vulnerabilities like SQL injection, XSS, IDOR, authentication issues, file upload bugs, and privilege escalation.
5. Practice legally
    If Hack The Box and TryHackMe are expensive, use free resources like:

* PortSwigger Web Security Academy
* OverTheWire
* websploit.org
* https://ai-cyber-training.com
* Hack The Box free machines
* TryHackMe free rooms
* YouTube channels like John Hammond, NetworkChuck, The Cyber Mentor, and LiveOverflow

Don’t worry too much about “new tools” at the start. Tools change, but fundamentals stay. Learn what the tools are doing behind the scenes. Once your basics are strong, tools will make much more sense.

To stay updated, follow:

* GitHub trending security tools, like the ones in this GitHub repository 
* Hacker News
* The Hacker News
* BleepingComputer
* Security Twitter/X accounts
* Reddit communities like r/cybersecurity, r/netsec, and r/HowToHack
* Exploit-DB
* OWASP
* Kali Linux tools updates

My advice: make a simple weekly plan. For example, spend one month on networking and Linux, one month on web basics, then start web security with PortSwigger Academy. Take notes, practice daily, and build small projects/labs.

You don’t need expensive platforms to start. You need consistency, curiosity, and strong fundamentals. Keep going... you’re already on the right path.

**Comment by akeem-eng** [comment]
check this links https://www.qualys.com/training
https://www.cbtnuggets.com/it-training/(isc)2/isc-2-cissp
www.isaca.com

**Comment by johnybald** [comment]
Can someone help me hack a messenger account
